docs/ems_reporting.md

```diff
@@ -0,0 +1,35 @@
+# EMS reporting
+
+Cumulus reports usage statistics to the [ESDIS Metrics System (EMS)](https://earthdata.nasa.gov/about/science-system-description/eosdis-components/esdis-metrics-system-ems).
+Two types of reports are generated: ingest and distribution.
+
+## Ingest
+
+Cumulus creates three ingest related reports to EMS, they are Ingest, Archive and Archive Delete reports.
```

Comment:
reword to 'Cumulus creates three ingest related reports for EMS: Ingest, Archive and Archive Delete.'